app/models/early_career_payments/eligibility.rb

```diff
@@ -38,11 +38,11 @@ class Eligibility < ApplicationRecord
     }, _prefix: :itt_subject
 
     enum itt_academic_year: {
-      "2018_2019": 0,
-      "2019_2020": 1,
-      "2020_2021": 2,
-      none_of_the_above: 3
-    }, _prefix: :itt_academic_year
+      AcademicYear.new(2018) => AcademicYear::Type.new.serialize(AcademicYear.new(2018)),
```

Comment:
@nomasprime Does this create an integer to work with AR enums?

Reply:
No, string enum.

Reply:
@nomasprime Ah, yes, a while since I have used Enums, completely forgot about being able to also use strings. Makes sense